client/state/current-user/email-verification/actions.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+	EMAIL_VERIFY_STATE_RESET,
+} from 'state/action-types';
+import wpcom from 'lib/wp';
+
+export function verifyEmail() {
```

Comment:
Sure thing - I'll dig in to it! I just found out about the data layer the other day on my analytics PR, 🎉